code/game/machinery/newscaster.dm

```diff
@@ -1067,6 +1067,8 @@ var/list/obj/machinery/newscaster/allCasters = list() //Global list that will co
 	var/scribble_page = null
 
 /obj/item/weapon/newspaper/attack_self(var/mob/user)
+	if (stat != CONSCIOUS)
```

Comment:
Incapacitated is better in this case, since it checks this and also stunned

Reply:
yep